### DISCUSSION TITLE
No hooks logs in Visual Studio with Husky.Net

### DISCUSSION THREAD

**Discussion by NilkOne** [alirezanet/Husky.Net#51] [Q&A]
### Version

0.5.1

### Details

In Visual Studio Enterprise 2022 (64 bits) - Preview Version 17.3.0 Preview 6.0,
I can't view hook error logs.

I can view an error message "One or more errors occurs" but no details when comiting.
No more details in Output window (Git source control section).

I can view logs only if I run a git command line in dos/powershell.

(if I copy the hook directly in Git Hooks folder, that work and I can see error message in VS window)

Any solutions ?

![image](https://user-images.githubusercontent.com/18186517/183901704-fa1e5f9e-ba3e-4dcb-b486-d63ba742a2d6.png)

### Steps to reproduce

- open a VS solution with git hooks
- try a bad commit
- no more information about errors

**Comment by alirezanet** [comment]
Hi @NilkOne,

First of all, I don't have this problem in Visual Studio 2022 v17.0.6.  for me, it shows the error message directly!
![image](https://user-images.githubusercontent.com/7004080/183904341-2f09787e-3d5a-448a-9b84-f7f19d40cb17.png)
maybe if update your visual studio it will fix.

second, if the visual studio doesn't provide any way to check git error messages I can not do much about that in husky.

**Comment by NilkOne** [comment]
Thanks for your rapid answer

Can you share your hooks please ?

**Comment by alirezanet** [comment]
> Can you share your hooks please ?

I've tested this problem with the current husky repo.

**Comment by alirezanet** [comment]
I moved this issue to discussions, feel free to ask any question if you couldn't manage to solve the problem.

**Reply by NilkOne** [reply]
Thanks

I've tried with https://github.com/alirezanet/Husky.Net repo

I needed to run theses commands : 

- dotnet husky install
- dotnet tool restore

In VS Professional 2022 (64 bits) - Version 17.2.6, same problem (I'm uptodate)

![image](https://user-images.githubusercontent.com/18186517/183910325-1cc37a8f-e8a4-4a8c-8593-5e02fc5023e0.png)

**Reply by alirezanet** [reply]
Weird! I'll do a little bit of research about this later.

**Reply by NilkOne** [reply]
thanks @alirezanet

**Reply by alirezanet** [reply]
![image](https://user-images.githubusercontent.com/7004080/183944273-4a3ff02d-2ab5-4b43-a351-11e7df2f214e.png)

I've tested this on my laptop and also updated my visual studio to 17.3.0 but still, I'm getting the error messages! could be related to your visual studio language, can you try to use English?

**Reply by NilkOne** [reply]
Hello,
I tried with English language in 17.3.0 version on Windows 10 : same problem
Do you test on a clean clone folder of the project ?

Because when I clone the project, I need to install Husky otherwise I can commit with 'test' commit message.


![image](https://user-images.githubusercontent.com/18186517/184100886-286b89b0-3da9-49b8-8bb6-78f12c9e3f68.png)

I tested all logging verbosity but nothing change

![image](https://user-images.githubusercontent.com/18186517/184101698-821bd22f-773b-453d-a49d-3cc34c3f3371.png)

**Reply by alirezanet** [reply]
Yes, after cloning the project you either must install the husky manually or build and restore the project (it will automatically install the husky). 
I'm still looking for a way to simulate the problem and hopefully find a solution for you.

**Reply by NilkOne** [reply]
OK thanks
